ggml/src/ggml-cpu/ggml-cpu.c

```diff
@@ -3449,6 +3469,14 @@ int ggml_cpu_has_dotprod(void) {
 #endif
 }
 
+int ggml_cpu_has_fp16_va(void) {
+#if defined(__ARM_FEATURE_FP16_VECTOR_ARITHMETIC)
```

Comment:
I wonder why we do a second ifdef here when this variable is set to 0 or 1 elsewhere.

Reply:
See my comment below, case 1. The way I see it, during runtime detection the host may report that the CPU supports this feature, but if we disabled it at compilation, we want the function to always return 0.